Pidgin/Parser.NoneOf.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Linq;
+using Pidgin.ParseStates;
+
+namespace Pidgin
+{
+    public static partial class Parser
+    {
+        /// <summary>
+        /// Creates a parser which parses and returns a character as long as it does not match one of the specified characters.
+        /// </summary>
+        /// <param name="chars">A sequence of characters that should not be matched</param>
+        /// <returns>A parser which parses and returns a character that does not match one of the specified characters</returns>
+        public static Parser<char, char> NoneOf(params char[] chars)
```

Comment:
Let's go with `AnyCharExcept`. It leaves `NoneOf` free for `Not(OneOf(...))`, which is how I would expect a parser named `NoneOf` to be implemented.

I think it would be a good idea to copy the input array, so as to safeguard against it getting mutated.